lib/rubocop/cop/registry.rb

```diff
@@ -117,9 +117,18 @@ def length
         @registry.size
       end
 
-      def enabled(config, only)
+      def enabled(config, only, only_safe = false)
```

Comment:
I thought we had a rule not to do optional boolean params. In general these days we should favour keyword params all the time.

Reply:
We could write an internal affairs cop to enforce this. Maybe it would be useful to have it public, but disabled by default. The `expect_offense` spec helpers takes in an optional argument for a file name. I'm not sure if we have other usages of optional arguments.

Reply:
I think such a cop should probably be public.